go.mod

```diff
@@ -49,9 +49,7 @@ require (
 	sigs.k8s.io/kind v0.20.0
 )
 
-retract (
-	v0.6.0
-)
+retract v0.6.0
```

Comment:
no need to change this

Reply:
This happened automatically with `go mod tidy`. I can roll it back.